scp/values.yaml

```diff
@@ -31,59 +77,147 @@ abacus:
 access-pep:
   enabled: true
   name: access-pep
+  existingImagePullSecret: *scpImagePullSecretName
   image:
     repo: ghcr.io/virtru-corp/access-pep/access-pep
     tag: sha-cac697c
-  imageCredentials:
-    registry: ghcr.io
-    username: overrideme
-    password: overrideme
-    email: overrideme
   config:
-    disableTracing: "true"
-    attrAuthorityHost: "http://attributes:4020"
+    disableTracing: *disableTracing
+    attrAuthorityHost: *attrEndpoint
     entitlementPdpHost: "http://entitlement-pdp:3355"
+    keycloakAttrAuthorityClientID: "tdf-client"
+    keycloakAttrAuthorityClientSecret:
+
+# Attribute Chart Overrides
+attributes:
+  fullnameOverride: attributes
+  secretRef: |-
+    name: scp-attributes-secret
 
-backend:
+# Configuration Chart Overrides
+configuration:
   enabled: true
-  entitlement-pdp:
-    opaConfig:
-      policy:
-        useStaticPolicy: false
-        allowInsecureTLS: true
-        OCIRegistryUrl: "https://scp-docker-registry:5000"
-        # -- Resource path to use to download bundle from configured service
-        bundleRepo: "scp-docker-registry:5000/entitlements-policybundle"
-        bundleTag: 0.0.2
-  entity-resolution:
-    config:
-      disableTracing: "true"
-  keycloak:
-    extraEnv: |-
-      - name: CLAIMS_URL
-        value: http://entitlement-pdp:3355/entitlements
-      - name: JAVA_OPTS_APPEND
-        value: -Djgroups.dns.query={{ include "keycloak.fullname" . }}-headless
-      - name: KC_DB
-        value: postgres
-      - name: KC_DB_URL_PORT
-        value: "5432"
-      - name: KC_LOG_LEVEL
-        value: INFO
-      - name: KC_HOSTNAME_STRICT
-        value: "false"
-      - name: KC_HOSTNAME_STRICT_BACKCHANNEL
-        value: "false"
-      - name: KC_HOSTNAME_STRICT_HTTPS
-        value: "false"
-      - name: KC_HOSTNAME_URL
-        value: {{ ( include "backend.keycloak.externalUrl" . ) | quote }}
-      - name: KC_HOSTNAME_ADMIN_URL
-        value: {{ ( include "backend.keycloak.externalUrl" . ) | quote }}
-      - name: KC_HTTP_ENABLED
-        value: "true"
-      - name: KC_FEATURES
-        value: "preview,token-exchange"
+  fullnameOverride: configuration
+  server:
+    image:
+      tag: 0.3.5
+    secretRef: |-
+      name: scp-configsvc
+    postgres:
+      host: postgresql
+      # Override with value
+      password:
+    imagePullSecrets: *imagePullSecrets
+
+# Entitlement Policy Bootstrap
+entitlement-policy-bootstrap:
+  policyGlobPattern: "configs/fed-demo/entitlement-policy/*"
+  bundleRepo: *entitlementPolicyBundleRepo
```

Comment:
Richard is creating a new dedicated entitlement policy repo for fed.  Is this the one you will use?  https://github.com/virtru-corp/entitlement-policy-federal

Reply:
That is where I sourced my current install config from. The way this chart is currently setup is those files sit in some place relative to the installation dir (in this case /configs/fed-demo/entitlement-policy ) and they are picked up by the chart to bundle and push to the repo